### PR TITLE
Use /p to set the inputchanges path

### PR DESCRIPTION
**Pull request by norahuang**
The QTest SDK need to aware the existence of the input changes passed from GBR. Change /InputChanges command line option to a environment parameter /p:[Sdk.BuildXL]inputChanges

### CODE REVIEW

File: Public/Src/Engine/Scheduler/Scheduler.cs [hunk 1]
```diff
@@ -4915,11 +4920,18 @@ private void ProcessFileChanges(LoggingContext loggingContext, SchedulerState sc
         {
             InputChangeList inputChangeList = null;
 
-            if (m_configuration.Schedule.InputChanges.IsValid)
+            var inputChangesFilePath = Environment.GetEnvironmentVariable("[Sdk.BuildXL]inputChanges");
```
Comment (mpysson) [inline Public/Src/Engine/Scheduler/Scheduler.cs]: >var inputChangesFilePath = Environment.GetEnvironmentVariable("[Sdk.BuildXL]inputChanges"); [](start = 12, length = 91)

The ability to consume an externally specified changes is something that's potentially useful to BuildXL beyond this code coverage feature. I don't really want to couple it to the QTest sdk. It's also a bit sneaky to toggle the BuildXL behavior just based on an environment variable. I think we should preserve the command line configuration.

File: Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc [hunk 2]
```diff
@@ -126,7 +130,7 @@ export function runQTest(args: QTestArguments): Result {
      
     let changeAffectedInputListWrittenFile = undefined;
     let changeAffectedInputListWrittenFileArg = {};
-    if (qCodeCoverageEnumType === "DynamicCodeCov"){
+    if (qCodeCoverageEnumType === "DynamicCodeCov" && File.exists(qInputChangesFilePath)){
```
Comment (mpysson) [inline Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc]: >&& File.exists(qInputChangesFilePath) [](start = 51, length = 37)

why is this necessary? Don't we want the ability to run code coverage without the inputChangesFile being set?

Comment (norahuang) [reply Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc]: I need to know if it is a change based code coverage or a full code coverage. I wanted to use the existence of the inputChanges to distinguish these two cases. Without knowing the full code coverage doesn't work. There is another approach which uses a separated /p: to indicated the change based one. This will make the code coverage feature separated from input changes. But the downside is introducing a extra option.

**Comment by norahuang** [comment]
Close this PR with another which adds a extra /p: instead of changing /InputChanges to /p:
